WPMedia/fusion-news-theme-blocks
# Tmedia-35/correct font usage

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
This PR addresses incorrect fonts being used in some blocks. Corrects the font in most blocks, the Gallery is addressed in a related pull request: [engine-theme-sdk pull request](https://github.com/WPMedia/engine-theme-sdk/pull/352/files)

## Jira Ticket
- [TMEDIA-35](https://arcpublishing.atlassian.net/browse/TMEDIA-35)

## Acceptance Criteria
PrimaryFont being used in the following blocks:

- Byline
- Gallery + video labels
- See More button text
- Pull Quote and Block Quote attribution text
- Tags Bar titles
- Footer copyright text + footer section headings

SecondaryFont being used in the following blocks:
- Pull Quote and Block Quote text
- 
## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-35/correct_font_usage`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block,@wpmedia/byline-block,@wpmedia/search-results-list-block,@wpmedia/results-list-block,@wpmedia/card-list-block,@wpmedia/footer-block,@wpmedia/article-tag-block,@wpmedia/shared-styles`
3. Go to http://localhost/pf/all-blocks/?_website=arc-demo-1
4. Observe the following blocks are now using the PrimaryFont used throughout the site for the following blocks:
- Byline
- See More button text(search for text `Ver más`)
- Pull Quote and Block Quote attribution text
- Tags Bar titles
- Footer copyright text + footer section headings 
5. Observe SecondaryFont being used in Pull quote and Block quote text
5. For Gallery + video labels, go to http://localhost/pf/gallery-video-label-test/?_website=the-gazette and observe the correct styles for Gallery and Video label. Gallery controls are part of another PR: https://github.com/WPMedia/engine-theme-sdk/pull/352

## Effect Of Changes
### Before
Fonts default to secondary or global font on certain blocks

### After
All fonts are correct as outlined above

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects
N/A

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
